interceptors/GoogleTagManagerLayoutInterceptor.cfc

```diff
@@ -5,49 +5,53 @@ component extends="coldbox.system.Interceptor" {
 
 	public void function configure() {}
 
-	public void function postLayoutRender( event, interceptData={} ) {
-		var layout = Trim( interceptData.layout ?: "" );
+	public void function postRenderDelayedViewlets( event, interceptData={} ) {
+		var layout = Trim( Event.getCurrentLayout() ?: "" );
```

Comment:
This whole block can & should just be replaced with:

```cfc
var layout = ListFirst( event.getCurrentLayout(), "." );
```

Note also the use of elvis here is not necessary and won't work.

Reply:
Modification applied thanks!